### DISCUSSION TITLE
Members profile and managing members

### DISCUSSION THREAD

**Discussion by niloc7** [Team-Tea-Time/laravel-forum#371] [Ideas]
Hey there!

Was wondering if there's something I'm missing from the demo site, or if it's not yet available, in regards to Member Profile as well as managing users (deleting, adding users, etc) for an admin.

Thanks!

**Comment by Riari** [comment]
Hi! Features like those are intentionally omitted because they're not specific to forums - they're standalone features that can (and should!) be implemented independently, either directly in the host application or via other packages.

I've considered adding user profiles as an optional feature in the past as I know forum user profiles tend to focus heavily on forum content, but it's pretty easy to fetch that kind of data with the models or REST API. Plus it would be another thing for me to maintain and I don't really have the time for that!

**Reply by niloc7** [reply]
I see, kinda like a starter pack / headless application, to incorporate whatever the user wants to do with it. 

Ok will head towards that direction! Thanks! :)
